ksqldb-rest-model/src/test/java/io/confluent/ksql/rest/entity/SchemaInfoTest.java

```diff
@@ -0,0 +1,24 @@
+package io.confluent.ksql.rest.entity;
+
+import com.google.common.collect.ImmutableMap;
+import io.confluent.ksql.schema.ksql.types.SqlBaseType;
+import org.junit.Test;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+
+public class SchemaInfoTest {
+
+  @Test
+  public void shouldCorrectlyFormatDecimals() {
```

Comment:
I wasn't able to cast a Decimal column without either, was going to ask you about this? It might be because the factory I've found doesn't implement these as optional? I'll see about passing the nulls